tests/virtualization/universal/waitfor_guests.pm

```diff
@@ -60,10 +59,12 @@ sub add_pci_bridge {
         # Skip if a pcie-to-pci-bridge is already present
         return if (script_run("cat $xml | grep 'controller' | grep 'pcie-to-pci-bridge'") == 0);
 
-        my $regex = '<controller type=.pci. .*model=.pcie-root..*>';
-        my $addbefore = '<controller type="pci" model="pcie-root-port"/><controller type="pci" model="pcie-to-pci-bridge"/>';
-        # the add_before.sh script inserts a given line before the line matching the given regex
-        assert_script_run("virsh dumpxml $guest | /root/add_before.sh '$regex' '$addbefore' > $xml");
+        # Add 10 pcie-root-port devices and a pcie-to-pci brdige
+        assert_script_run("virsh dumpxml $guest > $xml");
+        for (1 .. 10) {
```

Comment:
@tbaev  I am curious about the minimum required pcie-root-port for our hotplugging test and how you decide to add 10 ports?